### ISSUE TITLE
Notebook test is broken at head.

### ISSUE THREAD

**Issue by dabacon** [quantumlib/Cirq#4081]
See PR #4080  The Notebook Tests against PR test fails for a PR with no real change in the PR.

```
Exception encountered at "In [13]":
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
/opt/hostedtoolcache/Python/3.7.10/x64/lib/python3.7/site-packages/ipykernel/pylab/backend_inline.py in configure_once(*args)
    167             def configure_once(*args):
    168                 activate_matplotlib(backend)
--> 169                 configure_inline_support(ip, backend)
    170                 ip.events.unregister('post_run_cell', configure_once)
    171             ip.events.register('post_run_cell', configure_once)

/opt/hostedtoolcache/Python/3.7.10/x64/lib/python3.7/site-packages/IPython/core/pylabtools.py in configure_inline_support(shell, backend)
    384     )
    385 
--> 386     from matplotlib_inline.backend_inline import configure_inline_support_orig
    387 
    388     configure_inline_support_orig(shell, backend)

ImportError: cannot import name 'configure_inline_support_orig' from 'matplotlib_inline.backend_inline' (/opt/hostedtoolcache/Python/3.7.10/x64/lib/python3.7/site-packages/matplotlib_inline/backend_inline.py)
```

**Comment by dabacon** [comment]
I suspect this is coming from IPython 7.23 see the release notes here
https://ipython.readthedocs.io/en/stable/whatsnew/version7.html

**Comment by dabacon** [comment]
Testing adding `matplotlib-inline` as a dependency in the requirement file for the test.

**Comment by dabacon** [comment]
> Testing adding `matplotlib-inline` as a dependency in the requirement file for the test.

Did not fix.

**Comment by balopat** [comment]
I opened https://github.com/ipython/ipython/issues/12941 - downgrade to ipython==7.22 fixes it, they released 7.23 yesterday, hence the breakage.  #4082 will fix it.

**Comment by balopat** [comment]
Some more color: from this discussion https://www.reddit.com/r/voidlinux/comments/n2nxyd/matplotlib_inline_broken_after_most_recent_update/gwky6c9/ and this PR (https://github.com/quantumlib/Cirq/pull/4082/files) it seems it's a bug...but we'll see, for now I think we can safely pin to 7.22.

**Comment by balopat** [comment]
This is now fixed in #4077.
